.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: weekly
+  ignore:
+    # Version 3 and higher required Java 11, so ignore those
+    - dependency-name: "caffeine"
+      versions: [ "[3.0.0)" ]
```

Comment:
`[3.0.0)` being the standard maven range for 3.0.0 and higher as per https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates#ignore